website/content/docs/concepts/domain-model/auth-methods.mdx

```diff
@@ -31,6 +31,98 @@ The password auth method has the following additional attributes:
+
+- `user_attr` - (optional) If set, defines the attribute on a user's entry 
+  matching the login-name passed when the user authenticates.  Examples: cn, uid  
+
+- `user_filter` - (optional) If set, the go template used to construct an LDAP
+  user search filter. The template can access the following context variables:
+  [UserAttr, Username]. The default user_filter is ({{.UserAttr}}={{.Username}})
+  or (userPrincipalName={{.Username}}@UPNDomain) if the upn-domain parameter is
+  set.
+
+- `enable_groups` - (optional) If true, an authenticated user's groups will be
+  found during authentication. Defaults to false.
+
+- `group_dn` - (optional) If set, the base DN under which to perform a group
+  search. Example: ou=Groups,dc=example,dc=com
+
+  Note: there is no default, so no base dn will be used for group searches if
+  it's not specified
+
+- `group_attr` - (optional) If set, the LDAP attribute to follow on objects
+  returned by group_filter in order to enumerate user group membership.
+  Examples: for group_filter queries returning group objects, use: cn. For
+  queries returning user objects, use: memberOf. The default is cn.
+
+- `group_filter` - (optional) If set, the Go template used when constructing the
+  group membership query. The template can access the following context
+  variables: [UserDN, Username]. The default is
+  (|(memberUid={{.Username}})(member={{.UserDN}})(uniqueMember={{.UserDN}})),
+  which is compatible with several common directory schemas.
+
+- `certificates` - (optional) If set, PEM encoded x509 certificates in ASN.1
+  DER form that can be used as trust anchors when connecting to an LDAP
+  provider. 
+
+- `client_certificate` - (optional) If set, a PEM encoded x509 certificate in
+  ASN.1 DER form to be used as a client certificate. It must be set if the
+  optional client_certificate_key is specified.
+
+- `client_certificate_key` - (optional) If set, a PEM encoded certificate key in
+  PKCS #8, ASN.1 DER form. It must be set if the optional client_certificate is 
+  specified.
+  
+- `bind_dn` - (optional) If set, the distinguished name of entry to bind when 
+  performing user and group searches. Example:
+  cn=vault,ou=Users,dc=example,dc=com 
+
+- `bind_password` - (optional) If set, the password to use along with bind_dn
+  when performing user search. It must be set if the optional bind_dn is
+  specified. 
+
+- `use_token_groups` - (optional) If true, use the Active Directory tokenGroups
+  constructed attribute of the user to find the group memberships. This will
+  find all security groups including nested ones.
+
+- `account_attribute_maps` - (optional) If set, the attribute maps from custom
+  attributes to the standard account attributes of fullname and email.  These
+  maps are represented as key=value where the key equals the from_attribute and
+  the value equals the to_attribute.  For example "preferredName=fullName".  All
+  attribute names are case insensitive.
+  
+
```

Comment:
Probably want to get @Dan-Heath to have a look at these

Reply:
Added @Dan-Heath as a reviewer.